account_partner_aged_statement_webkit/report/partner_aged_statement.mako

```diff
@@ -0,0 +1,174 @@
+## -*- coding: utf-8 -*-
+<!DOCTYPE html SYSTEM
+"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <style type="text/css">
+            .total{
+               font-weight:bold;
+            }
+            .break{
+                display: block;
+                clear: both;
+                page-break-after: always;
+            }
+
+            ${css}
+        </style>
+    </head>
+
+    <%!
+    def amount(text):
+        # replace by a non-breaking hyphen (it will not word-wrap between hyphen and numbers)
+        return text.replace('-', '&#8209;')
+
+    %>
+    <body>
+    %for idx, partner in enumerate(objects):
+        <%setLang(partner.lang)%>
+        %if idx > 0:
+            <div class="break"></div>
+        %endif
+        <br/>
+        <br/>
+        <%from datetime import date %>
+        ${_('Date')}: ${formatLang(str(date.today()), date=True)}
+        <br>
+        ${_('Partner')}: ${partner.name}
+        <br>
+        ${_('Subject')}: <b>${_('Overdue Statement')}</b>
+        <br/>
+        <br/>
+        %if show_message:
+            %for message_line in message(partner, company):
+                <p>
+                ${message_line}
+                </p>
+            %endfor
+        %endif
+        <br>
+        ${user.name}
+        <br>
+        <br>
+        %if (partner.credit + partner.debit == 0) :
+            <div class="title">${_('Nothing due for this partner')}</div>
+        %else:
+            <div class="title">${_('Aged Balance')}</div>
+            <br>
+        %if get_balance(partner, company):
```

Comment:
Bad indentation, this if is nested within the else above. It might be misleading.